cfpb/consumerfinance.gov
# Wagtail: Add Simplified Chinese to language dropdown

## Additions

-  Add Simplified Chinese to language dropdown to Wagtail.
## How to test this PR

1. Pull branch, run migrations.
2. Edit or create a page in wagtail and go to the configuration tab and see that the language drop-down now has Chinese (Siimplified) as an option.


## Screenshots

<img width="731" alt="Screen Shot 2021-06-09 at 3 35 56 PM" src="https://user-images.githubusercontent.com/704760/121417927-6e406e80-c938-11eb-993f-6b795c5c7d2f.png">


## Notes and todos

- This is part one of a multi-part update. After this goes in all the chinese pages will be changed to "Chinese (Simplified)" and then "Chinese" will be updated to "Chinese (Traditional)". Then the chinese pages will again be updated to be either traditional or simplified.
